src/util/prom_types.py

```diff
@@ -7,13 +7,17 @@
 PROM_HEADERS = ''
 PROM_QUERY_INTERVAL = 300
 PROM_QUERY_STEP = 3
+PROM_QUERY_START_TIME = '2023-11-13T02:47:57Z'
```

Comment:
should not be static. We might define it based on the running time like below or set to '' to be handled if empty later. 

        PROM_QUERY_END_TIME = datetime.datetime.now()
        PROM_QUERY_START_TIME = PROM_QUERY_END_TIME - datetime.timedelta(seconds=PROM_QUERY_INTERVAL)

Reply:
Ok @sunya-ch , I have set it to empty. Hope this is also a valid setting.